charon-ml/src/LlbcAstUtils.ml

```diff
@@ -1,6 +1,18 @@
 include GAstUtils
 open LlbcAst
 open Utils
+open Collections
+
+let fun_decl_list_from_crate (crate : crate) : fun_decl list =
+  snd (List.split (FunDeclId.Map.bindings crate.fun_decls))
+
+let get_fun_args (fun_decl : fun_decl) : var list =
```

Comment:
Please document what this function does

Reply:
Could you also clarify the docs of `GExprBody` so we know in which order the arguments go in `locals`?